code/game/objects/items/storage/wallets.dm

```diff
@@ -64,6 +64,12 @@
 		var/obj/item/card/id/id_card = card
 		if(!istype(id_card))
 			continue
+
+		// Chameleon IDs explicitly jump to the front so they can disguise other cards in wallets.
+		if(istype(id_card, /obj/item/card/id/advanced/chameleon))
```

Comment:
I thought  `istype` checks were frowned upon

Reply:
It's inevitable in this scenario. Wallets can hold items other than ID cards, so I'll need some sort of typed loop.

I can replement this with a typed for loop, or an `as anything` for loop as I'd still need istype checks.

Reply:
I mean, did you consider marking the functionality of "this ID goes to the front of the wallet" in some way other than behaviour assigned to a type? A trait?

Reply:
I didn't consider that. I could add it as a trait.